tools/provision.sh

```diff
@@ -15,17 +15,6 @@ AUTH_METHOD_NONE="none"
 # default to TLS for now
 AUTH_METHOD="${1:-$AUTH_METHOD_TLS}"
 
-COMPOSER_CONFIG="/etc/osbuild-composer/osbuild-composer.toml"
-
-# Path to a file with additional configuration for composer.
-# The content of this file will be appended to the default configuration.
-EXTRA_COMPOSER_CONFIG="${2:-}"
-
-if [[ -n "${EXTRA_COMPOSER_CONFIG}" && ! -f "${EXTRA_COMPOSER_CONFIG}" ]]; then
-    echo "ERROR: File '${EXTRA_COMPOSER_CONFIG}' with extra configuration for composer does not exist."
-    exit 1
-fi
```

Comment:
The Weldr API test case depens on the changes in `provision.sh`: https://github.com/osbuild/osbuild-composer/blob/073e304978acade8bfa059f00a005402aa037c99/test/cases/weldr-distro-dot-notation-and-aliases.sh#L34-L43

Reply:
Thanks for the review and for noticing :heart_eyes: 
Dropped those changes.